docker-compose.yml

```diff
@@ -245,7 +245,7 @@ services:
     restart: unless-stopped
     image: docker.io/jaegertracing/jaeger:2.4.0
     volumes:
-      - ./jaeger.yml:/etc/jaeger/config.yml
+      - ./monitoring/jaeger.yml:/etc/jaeger/config.yml
```

Comment:
I think that merging main after the other PR was merged caused the jaeger.yml to appear again, can you remove it?

Reply:
This should be there though, the configuration files are moved into the `monitoring/` folder, unless I'm missing something. However we should consider if we want those files in that folder, I'm open to other suggestions, my main consideration was I didn't want to clutter the root folder with more monitoring configuration files.

Reply:
What I meant is that there is currently both a jaeger.yml and a monitoring/jaeger.yml. The delete part of the move was probably undone because of the other PR, so jaeger.yml needs to be deleted in this PR.

Reply:
Ah got it!